src/gate/index.js

```diff
@@ -1,4 +1,7 @@
-var BloomFilter = require('bloom-filter-remixed');
+const BloomFilter = require('bloom-filter-remixed');
+const sha256 = require('sha.js/sha256')
```

Comment:
Could we add some comments here on the use of this library? e.g. isomorphic, synchronous, trusted because it's part of [Browserify's polyfill](https://github.com/crypto-browserify/crypto-browserify) (maybe say `crypto-browserify -> createHash -> sha.js`) but this particular module and submodule for smallest bundle size.

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/7e6dfa937363205c20409112b4cd4f9e5553baf0